src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -1064,15 +1003,20 @@ public void onProgressChanged(long numBytes,
 				String ioExceptionString = "zipFiles: problem closing Inputstream content for file "
 						+ fileName + " " + ioException;
```

Comment:
These 4 lines could be put in a method whose argument is the text and a lot of duplicate code would be eliminated.